# Set a camera parameter

CameraParam param

---

